src/app/content/hooks/locationChange/resolveContent.ts

```diff
@@ -67,16 +67,26 @@ const resolveBookReference = async(
   match: Match<typeof content>
 ): Promise<[string, string, string | undefined]> => {
   const state = getState();
-  const bookSlug = match.params.book;
   const currentBook = select.book(state);
 
+  if (!match.params.uuid && !match.params.book) {
+    throw new Error('Nor a book nor a uuid found');
```

Comment:
if you do the union thing, you shouldn't have to handle this case because ts will figure out narrowing the union type. probably.